sap_hana/assets/configuration/spec.yaml

```diff
@@ -75,3 +75,8 @@ files:
           type: boolean
       - template: instances/tls
       - template: instances/default
+      - name: tenant_databases_schema
```

Comment:
What do you think about changing this name to be something like "schema", since to users it's just the schema they are collecting metrics from?

Reply:
That's what I wanted to do at first but then I realised that we have in fact two different schemas in the code `SYS` and `SYS_DATABASES`. 

https://github.com/DataDog/integrations-core/blob/master/sap_hana/datadog_checks/sap_hana/queries.py#L37
https://github.com/DataDog/integrations-core/blob/master/sap_hana/datadog_checks/sap_hana/queries.py#L80

However, `schema` is also a good option from the user point of view, and it's way easier. What do you think?

Reply:
On a second thought, I updated the option to `schema` as you suggested! Let me know what you think

Reply:
Awesome! I agree, I think it's a lot more clear and the `SYS` schema is only used once.